admin_manual/release_notes.rst

```diff
@@ -6,7 +6,12 @@ ownCloud |version| Release Notes
 Changes in 9.0
 --------------
 
-Home folder rule is enforced in the user_ldap application in new ownCloud installations; see :doc:`configuration_user/user_auth_ldap`. This affects ownCloud 8.0.10, 8.1.5 and 8.2.0 and up.
+Home folder rule is enforced in the user_ldap application in new ownCloud installations; see
+:doc:`configuration_user/user_auth_ldap`. This affects ownCloud 8.0.10, 8.1.5 and 8.2.0 and up.
+
+The CalDAV and CardDAV backends where merged into core. If you where using the Calendar and/or
+Contacts app you need to run the ``dav:migrate-calendars`` and/or ``dav:migrate-addressbooks``
```

Comment:
afaik dav:migrate-calendars needs a user and we should probably add the full command